src/encoded/types/biosample.py

```diff
@@ -119,6 +119,89 @@ def treatments_summary(self, request, treatments=None):
             return ' and '.join(sorted([t for t in treat_list if t]))
         return 'None'
 
+    def _get_sample_tissue_organ(self, request, tissue_id):
+        tissue = None
+        organ_system = []
+        tissue_term = self._get_item_info(request, [tissue_id], 'ontology_terms')[0]  # 1 item list
+        if tissue_term:
+            tissue = tissue_term.get('display_title')
+            if 'slim_terms' in tissue_term:
+                slim_terms = self._get_item_info(request, tissue_term.get('slim_terms'), 'ontology_terms')
+                for st in slim_terms:
+                    if st.get('is_slim_for') in ['developmental', 'system', 'organ']:
+                        organ_system.append(st.get('display_title'))
+        return tissue, organ_system
+
+    def _get_item_info(self, request, item, itype):
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6380012)